elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -327,7 +343,16 @@ class PfeAccordion extends PFElement {
       requestAnimationFrame(() => {
         requestAnimationFrame(() => {
           panel.style.height = `${end}px`;
-          panel.addEventListener("transitionend", this._transitionEndHandler);
+
+          this._animate._tId = setTimeout(() => {
```

Comment:
The transitionend event was not firing on accordions that were inside of not-yet active tabsets.  This fixes that bug by adding an escape hatch of 2s if the event has not fired.